var/spack/repos/builtin/packages/py-prov/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyProv(PythonPackage):
+    """A Python library for W3C Provenance Data Model (PROV).
+
+    A library for W3C Provenance Data Model supporting PROV-JSON, PROV-XML and
+    PROV-O (RDF)
+    """
+
+    homepage = "prov.readthedocs.io/"
+    pypi     = "prov/prov-2.0.0.tar.gz"
+
+    version('2.0.0', sha256='b6438f2195ecb9f6e8279b58971e02bc51814599b5d5383366eef91d867422ee')
+
+    variant('dot', default=False, description='Graphical visualisation support for prov.model')
+
+    depends_on('python@3.6:4.0', type=('build', 'run'))
```

Comment:
```suggestion
    depends_on('python@3.6:3.999', type=('build', 'run'))
```
Version ranges are inclusive